gio/src/auto/flags.rs

```diff
@@ -2071,6 +2071,8 @@ impl ToValue for SubprocessFlags {
 bitflags! {
     #[doc(alias = "GTlsCertificateFlags")]
     pub struct TlsCertificateFlags: u32 {
+        #[doc(alias = "G_TLS_CERTIFICATE_FLAGS_NONE")]
+        const FLAGS_NONE = ffi::G_TLS_CERTIFICATE_FLAGS_NONE as u32;
```

Comment:
Can be hidden. This is the same as `TlsCertificateFlags::empty()`. It's useless C convenience